crates/agglayer-rpc/src/lib.rs

```diff
@@ -327,10 +328,56 @@ where
 
         Ok(())
     }
+
+    /// Verify the extra [`Certificate`] signature.
+    #[instrument(skip(self), level = "debug")]
+    pub(crate) fn verify_extra_cert_signature(
+        &self,
+        certificate: &Certificate,
+        maybe_extra_signer: Option<&H160>,
+        maybe_extra_signature: Option<Signature>,
```

Comment:
Suggestion (not strong opinion from my side): we haven't used `maybe` on other places where we have options AFAIK, for consistency I wouldn't use them here either. Comment also applies for other `maybe` variables in this PR.

Reply:
yeah true, I have renamed those